lms/static/scripts/frontend_apps/components/BookPicker.tsx

```diff
@@ -115,13 +125,23 @@ export default function BookPicker({
         />
       )}
       {step === 'select-chapter' && !error && (
-        <ChapterList
-          chapters={chapterList || []}
-          isLoading={!chapterList}
-          selectedChapter={chapter}
-          onSelectChapter={setChapter}
-          onUseChapter={confirmChapter}
-        />
+        <div
+          className={classnames(
+            // Set a preferred height for this content to give it more vertical
+            // room. Max-height rules on the containing Modal will prevent the
+            // Modal from exceeding available viewport space. Overflowing
+            // content will scroll.
+            'min-h-[25rem]'
```

Comment:
Achieves same results as the previous `LMS-Dialog--tall` CSS class and is more closely associated with the content it applies to.